src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Unix.cs

```diff
@@ -52,8 +68,18 @@ public string DriveFormat
             [SecuritySafeCritical]
             get
             {
-                Interop.libc.statfs data = Interop.libc.GetStatFsForDriveName(Name);
-                return Interop.libc.GetMountPointFsType(data);
+                string format;
+                int result = Interop.Sys.GetFormatInfoForMountPoint(Name, out format);
+                if (result == 0)
+                    return format;
+                else
+                {
+                    Interop.ErrorInfo err = Interop.Sys.ConvertErrorPlatformToPal(result).Info();
```

Comment:
`var err = new Interop.ErrorInfo(result);` would be nicer. 

The Info() thing is for Error.EFOO.Info() and the ConvertXxx functions shouldn't be needed directly.